README.md

```diff
@@ -161,6 +161,18 @@ To install zlib-ng system-wide using the configure script:
 make install
 ```
 
+Alternatively, you can build and install zlib-ng using [vcpkg](https://github.com/Microsoft/vcpkg/) dependency manager:
+
+```sh or powershell
+    git clone https://github.com/Microsoft/vcpkg.git
```

Comment:
Can you remove beginning whitespace from commands?

Reply:
Thanks a lot for your quick answer, I'll keep the format consistent with `zlib-ng`. :)